src/Package/Test/Repl/ReplCommandTest.cs

```diff
@@ -114,5 +117,39 @@ public class ReplCommandTest: IDisposable {
 
             _workflow.ActiveWindow.Dispose();
         }
+
+
+        [CompositeTest]
+        [Category.Repl]
+        [InlineData(false, "utf-8")]
+        [InlineData(true, "Windows-1252")]
+        public void SourceRScriptTest(bool echo, string encoding) {
+            var textBuffer = new TextBufferMock("", RContentTypeDefinition.ContentType);
+            var textView = new WpfTextViewMock(textBuffer);
+            var tracker = Substitute.For<IActiveWpfTextViewTracker>();
```

Comment:
Ah, I see, I can mock the factory for that when calling `GetOrCreateVisualComponent`